Sources/EdgeConstants.swift

```diff
@@ -117,11 +122,18 @@ enum EdgeConstants {
 
         enum Response {
             static let EVENT_HANDLE_TYPE_STORE = "state:store"
+            static let EVENT_HANDLE_TYPE_LOCATION_HINT = "locationHint:result"
 
             enum Error {
                 static let MESSAGE = "message"
                 static let NAMESPACE = "namespace"
             }
+
+            enum LocationHint {
+                static let SCOPE = "scope"
```

Comment:
Should we add the "locationHint" constant here only? 
static let KEY = "locationHint"

Reply:
Can you explain further? I guess I don't understand the question.

Reply:
It's fine. It is a nit point: Changing the location of constant.

Instead of Edge.LOCATION_HINT -> LocationHint.KEY